src/sdk/object_sdk.js

```diff
@@ -585,22 +593,26 @@ class ObjectSDK {
 
     async upload_multipart(params) {
         const ns = await this._get_bucket_namespace(params.bucket);
+        this.check_is_readonly_namespace(ns);
         if (params.copy_source) await this.fix_copy_source_params(params, ns);
         return ns.upload_multipart(params, this);
     }
 
     async list_multiparts(params) {
         const ns = await this._get_bucket_namespace(params.bucket);
+        this.check_is_readonly_namespace(ns);
```

Comment:
Do we need it on a list operation? 
Or, Can a read Only list multiparts?

Reply:
it's a list op on the `multiparts`, so I think it shouldn't be available on a read only resource,
but I would be happy to hear what the team thinks.

Reply:
I think we can remove the check here. access can be blocked on the source itself